core/src/consensus/tendermint/worker.rs

```diff
@@ -2121,3 +2113,46 @@ fn calculate_score(height: Height, view: View) -> U256 {
     let height = U256::from(height);
     u256_from_u128(std::u128::MAX) * height - view
 }
+
+struct MutTrigger<T> {
+    target: T,
+    deref_mut_triggered: Cell<bool>,
```

Comment:
How about adding some comments like "If the target is changed, deref_mut_triggered set true".

Reply:
`deref_mut` is the only way to get a mutable reference to the target. The comment has no additional information.